index.html

```diff
@@ -233,5 +233,14 @@ <h2 class="success__title">Изображение успешно загруже
     <div class="img-upload__message  img-upload__message--loading">Загружаем...</div>
   </template>
 
+  <script src="js/util.js"></script>
+  <script src="js/data.js"></script>
+  <script src="js/picture.js"></script>
+  <script src="js/preview-scale.js"></script>
+  <script src="js/preview-effect.js"></script>
+  <script src="js/form.js"></script>
+  <script src="js/movingSlider.js"></script>
+  <script src="js/show-picture.js"></script>
+  <script src="js/show-bigPicture.js"></script>
```

Comment:
```suggestion
  <script src="js/util.js"></script>
  <script src="js/data.js"></script>
  <script src="js/picture.js"></script>
  <script src="js/preview.js"></script>
  <script src="js/form.js"></script>
```
в preview.js объединям все что было в preview-scale preview-effect movingSlider
в picture.js объединяем все что было show-picture.js  и show-bigPicture.js